salvo/src/lib/test_source_tree.py

```diff
@@ -9,135 +9,428 @@
 from src.lib import (cmd_exec, source_tree)
 import api.source_pb2 as proto_source
 
+logging.basicConfig(level=logging.DEBUG)
 log = logging.getLogger(__name__)
 
+_GIT_TAG_LIST = """
```

Comment:
Moving the tag list into a global constant may be a regression. It introduces two problems:

1. The test cases are no longer "complete" or "all in one place" for the reader. The reader has to scroll up and down to understand them.
2. Multiple test cases use the same constant which to the reader implies that they all have to use the same constant / tag list. This likely isn't true and may confuse future maintainers.

How would you feel about moving this back into individual lists in the test cases that need this?

Reply:
Moved them back :)